tests/integration/test_plugins_dedup_batch4.py

```
@@ -128,6 +144,96 @@ def test_start_raises_plugin_load_error_on_oserror(self, tmp_path: Path) -> None
+                PluginExecutor,
+                "_readline_with_timeout",
+                side_effect=PluginError("startup read timed out"),
+            ),
+        ):
+            mock_proc = MagicMock()
+            mock_proc.stderr.read.return_value = b"startup stderr"
+            mock_popen.return_value = mock_proc
+
+            with pytest.raises(
+                PluginLoadError,
+                match="did not signal readiness.*startup stderr",
```

Ruff (0.15.21)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)